src/TableReader.jl

```diff
@@ -232,13 +238,14 @@ for (fname, delim) in [(:readdlm, nothing), (:readcsv, ','), (:readtsv, '\t')]
     push!(kwargs, Expr(:kw, :(skip::Integer), 0))  # skip::Integer = 0
     push!(kwargs, Expr(:kw, :(skipblank::Bool), true))  # skipblank::Bool = true
     push!(kwargs, Expr(:kw, :(colnames), nothing))  # colnames = nothing
+    push!(kwargs, Expr(:kw, :(normalizenames), false))  # skipblank::Bool = false
```

Comment:
Can you update the comment like `# normalizenames::Bool = false`?

Reply:
this is fixed in the latest commit